src/js/components/Layer/LayerContainer.js

```diff
@@ -66,8 +66,16 @@ const LayerContainer = forwardRef(
         if (modal && !element && anchorRef.current) {
           anchorRef.current.focus();
         }
+        if (
+          !modal &&
+          !element &&
+          anchorRef.current &&
+          (onEsc || onClickOutside)
```

Comment:
This doesn't seem to address the issue. If it's not a modal Layer, I don't think `onClickOutside` is connected up to anything.

Reply:
I gotcha. I was mistaken when testing. onEsc will work, however.

Reply:
ill wait on a response in #3771 before I make changes. Maybe we should just update the docs to be explicit about the purpose of non-modal layers instead.